x-pack/legacy/plugins/security/public/views/management/edit_role/index.js

```diff
@@ -88,7 +88,7 @@ const routeDefinition = (action) => ({
       return kfetch({ method: 'get', pathname: '/api/security/privileges', query: { includeActions: true } });
     },
     builtinESPrivileges() {
-      return kfetch({ method: 'get', pathname: '/api/security/v1/esPrivileges/builtin' });
+      return kfetch({ method: 'get', pathname: '/api/security/esPrivileges/builtin' });
```

Comment:
Yeah, we can do that for internal APIs, I was just waiting for a final decision on that ticket. Also how do you think we should treat OIDC and SAML callback routes - these had `v1` and are kind of external, but not like other `/api/*`?

Reply:
> Yeah, we can do that for internal APIs, I was just waiting for a final decision on that ticket
Good point :) We can leave this one alone for now.

> Also how do you think we should treat OIDC and SAML callback routes - these had v1 and are kind of external, but not like other /api/*?

Since end-users have to specify them in their IdP settings, I think we should start to consider them as being truly external. With that being said, we won't want to break backwards compatibility in a minor...